internal/config/profile.go

```diff
@@ -17,6 +17,7 @@ package config
 import (
```

Comment:
since you are touching this file what do you think about replacing `configHome()` with `os.UserConfigDir()`?

Reply:
configHome uses `os.UserHomeDir()` but it has extra logic to account for `XDG_CONFIG_HOME` (https://specifications.freedesktop.org/basedir-spec/latest/ar01s03.html)

Reply:
we can move it to this file as well if you want, at some point it made sense to keep them separate

Reply:
I have moved the function here

Reply:
I think I follow now, you want `os.UserHomeDir()` for anyone not in unix/posix

Reply:
Because if you follow the code it does something very similar in here https://cs.opensource.google/go/go/+/master:src/os/file.go;l=444-457?q=userconfigdir&ss=go%2Fgo

Reply:
also in case `XDG_CONFIG_HOME` is not set by some *nix not following the freedesktop spec

Reply:
oh sorry, you meant, [UserConfigDir](https://pkg.go.dev/os@go1.17#UserConfigDir) and I've been talking of [UserHomeDir](https://pkg.go.dev/os@go1.17#UserHomeDir)

We can't make the switch at this point, `UserConfigDir` was added in 1.13, we already had this logic in place (we started in 1.12) the current behaviour of `UserConfigDir` is different from ours so changing it would break macOS and Windows users